.gitignore

```diff
@@ -23,3 +23,6 @@
 # git-lfs files cannot be included in a template repository
 output/analysis/plots/gdp_educ.png
 output/paper/Template.pdf
+output/talk/TemplateSlides.pdf
+temp/TemplateHandout.pdf
```

Comment:
I think (?) everything in /temp should already be ignored per earlier lines in the `.gitignore`. If so we can remove this line; if not we should modify the `.gitignore` so everything in /temp is ignored!